pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -120,13 +141,61 @@ def __init__(
             zero_optimization: Enable ZeRO optimization. This is only compatible with precision=16. (default: True)
 
             stage: Different stages of the ZeRO Optimizer. 0 is disabled,
-                1 is optimizer state partitioning, 2 is optimizer+gradient state partitioning (default: 2)
+                1 is optimizer state partitioning, 2 is optimizer+gradient state partitioning,
+                3 is optimizer+gradient_parameter partitioning using the infinity engine (default: 2)
```

Comment:
Can you remove the defaults from the docstring? They are already included from the signature